include/psa/crypto.h

```diff
@@ -850,6 +850,15 @@ psa_status_t psa_export_public_key(psa_key_handle_t handle,
  * this function may be used to share a key with a different party,
  * subject to implementation-defined restrictions on key sharing.
  *
+ * The policy on the source key must have the usage flag
+ * #PSA_KEY_USAGE_COPY set.
+ * In addition, some lifetimes also require the source key to have the
+ * usage flag #PSA_KEY_USAGE_EXPORT, because otherwise the source key
```

Comment:
The wording isn't completely clear, for example:
* who is doing the "locking inside"? - presume this is the SE designer, but better to avoid unnamed actors in the documentation.
* Is the use case here one where a key is being copied out of the SE into a volatile key (for example), and the SE design defaults to denying this operation unless the key is extractable? - presumably a copy within the SE would only require COPY policy in this set up?

Reply:
I updated the documentation.